CONTRIBUTING.md

```diff
@@ -145,6 +145,14 @@ cases to our [automated test suite](ci/README.md) to guarantee enough
 coverage. A PR that makes significant code changes without contributing new test
 cases will be flagged by reviewers and will not be accepted.
 
+### Cherry-picks to release branches
```

Comment:
Should this be in Table of Contents?

Reply:
Thanks. I modified `hack/verify-toc.sh` and `hack/update-toc.sh` so that we auto-generate the TOC for CONTRIBUTING.md as well. Not sure why we didn't include it in the first place. This will avoid that kind of mistakes in the future.